Clear-ComputeNode.ps1

```diff
@@ -98,8 +80,8 @@ function Remove-NotUsedDockerNetworks {
     }
 }
 
-function Uninstall-Components {
-    Write-Host "Uninstalling components..."
+function Uninstall-MSIs {
```

Comment:
In my opinion previous name described what it is doing in general, this one describes implementation. I understand that You changed it because You added Uninstall-PythonPackages (which has the same problem I think). Maybe make separate functions for each component? Or just combine those two in one Uninstall-Components? Or You will have even better idea :)